src/utils/perfherder.js

```diff
@@ -117,7 +117,7 @@ const signaturesForPlatformSuite = async (seriesConfig) => {
         || (jobSignature.suite === jobSignature.test))
       ) {
         res[signatureHash] = {
```

Comment:
The `signatureHash` var name is misleading. Those should actually be signature ids.